src/components/shared/translations.js

```diff
@@ -85,6 +85,14 @@ const de = {
   obscure: {
     title: 'Dieser Text wurde als geschwärzt markiert, um Datenschutzrichtlinien zu entsprechen.'
   },
+  operations: {
```

Comment:
I think its a good Idea to open a section "operations". But then we should move `save` and `select` into here too.

Reply:
and we already have `cancel: "Abbrechen"`. We shouldn't double this with `abort: "Abbrechen"`.

Reply:
Thanks
Done in [chore (refs T34965): Add Trans-Keys to DpEditableList.](https://github.com/demos-europe/demosplan-ui/pull/605/commits/f2d17a472341e6446fbfeacacbc798d36019ec77)